playbooks/group_vars/ansibullbot.yml

```diff
@@ -21,7 +21,12 @@ ansibullbot_options:
   - skip_no_update_timeout
   - verbose
 
-ansibullbot_receiver_enabled: yes
+
+firewall_allowed_tcp_ports:
+  - 22
+  - 80
```

Comment:
@samdoran is this just for incoming traffic?

Reply:
Yes.